delete.py

```diff
@@ -1,17 +1,23 @@
 import sys
+import argparse
 
-from utils import run_shell_command, generate_heroku_app_name
+from .utils import run_shell_command, generate_heroku_app_name
 
 
-def delete_heroku(deployment_name):
+def delete(deployment_name, heroku_config={}):
```

Comment:
how about using `heroku_config=None` to be more explicit that the function did not get an arg v/s the situation where it was passed and empty dict?

Reply:
for describe func too
but otherwise this is good.

Reply:
also using a dictionary by default makes its mutable

Reply:
Got it! Thank you for the advice :)